rct229/ruletest_engine/ruletest_jsons/section6/rule_6_7.json

```diff
@@ -0,0 +1,394 @@
+{
+    "rule-6-7-a": {
+        "Section": 6,
+        "Rule": 7,
+        "Test": "a",
+        "test_description": "Proposed building is modeled with daylighting controls directly or through schedule adjustments.",
```

Comment:
Hey Yun Joon,

I noticed every test description reads the same way. If this is a consequence of the TCDs, then maybe you can update this to reflect the differences between the test cases. As it stands, I can't really tell what I should be looking for between the different test JSONs